lib/foreman_maintain/concerns/logger.rb

```diff
@@ -1,13 +0,0 @@
-require 'logger'
-
-module ForemanMaintain
-  module Concerns
-    module Logger
-      def logger
-        @logger ||= ::Logger.new($stderr).tap do |logger|
```

Comment:
You should be able to resolve the conflicts just by using `::Logger` when you are referring to the Ruby logger. I prefer `foreman_maintain/concerns/logger.rb` for defining the module to be included where needed, as this mimics the way people are used to organize this kind of modules in rails.